### PR TITLE
Remove brand color in organization settings

### PR DESCRIPTION
**Pull request by elenatorro**
Fixes [#4992](https://github.com/CartoDB/cartodb-management/issues/4992)

### CODE REVIEW

**Comment by javitonino** [comment]
Not so sure about Disqus removal, it's still in use -> https://github.com/CartoDB/cartodb-management/issues/4993#issuecomment-328893644

Let's wait for that discussion before going forward.

File: app/views/admin/organizations/settings.html.erb [hunk 1]
```diff
@@ -102,18 +88,6 @@
 
       <div class="FormAccount-row">
         <div class="FormAccount-rowLabel">
-          <label class="CDB-Text CDB-Size-medium is-semibold u-mainTextColor">Disqus shortname</label>
```
Comment (nobuti) [inline app/views/admin/organizations/settings.html.erb]: Let's revert this.

**Comment by Jesus89** [comment]
Deployed in ded10!

**Comment by Jesus89** [comment]
It works as expected :star2:

**Pull request merged**